riot/cli.py

```diff
@@ -111,6 +111,7 @@ def generate(ctx, recreate_venvs, skip_base_install, pythons, pattern):
 @RECREATE_VENVS_ARG
 @SKIP_BASE_INSTALL_ARG
 @click.option("--pass-env", "pass_env", is_flag=True, default=False)
+@click.option("--build", "build", is_flag=True, default=False)
```

Comment:
I wonder what the end goal "action" is.

Like, are we "building" the dev package, are we "installing" it, are we "rebuilding"/"reinstalling" it? Or are we recreating the venv (e.g. delete it and create from scratch)?

e.g. I think any of these names could work (`--build` is acceptable! just brainstorming):

- `--build`
- `--install`
- `--recreate`

Reply:
- `--force-build`/`--force-install` ?